azure/client.go

```diff
@@ -36,8 +37,14 @@ func NewAnonymousClient() Client {
 
 //NewClientFromPublishSettingsFile creates a new azure.Client and imports the publish
 //settings from the specified file path.
-func NewClientFromPublishSettingsFile(publishSettingsFilePath string) (Client, error) {
-	client := Client{}
+func NewClientFromPublishSettingsFile(publishSettingsFilePath string, serviceManagementURL ...string) (Client, error) {
+	var client Client
+	if len(serviceManagementURL) > 0 {
```

Comment:
Wasn't sure if was better to use a variadic parameter or create new functions altogether